docs/setup-cluster/deploy-cluster/installation-options.rst

```diff
@@ -1,20 +1,135 @@
-#############
- Basic Setup
-#############
+:orphan:
 
-Your training environment can be a local development machine, an on-premise GPU cluster, or cloud
-resources. To set up your training environment, follow the steps below.
+.. _installation-options:
 
-**************************************************
- Step 1 - Set ``DET_MASTER`` Environment Variable
-**************************************************
+######################
+ Installation Options
+######################
+
+In this guide, you'll find information about basic and advanced cluster configuration options, as
+well as additional installation options such as setting up clients, internet access, and firewall
+rules.
+
+.. _cluster-configuration:
+
+***********************
+ Configure the Cluster
+***********************
+
+Common configuration reference: :doc:`/reference/deploy/config/common-config-options`
+
+Master configuration reference: :doc:`/reference/deploy/config/master-config-reference`
+
+Agent configuration reference: :doc:`/reference/deploy/config/agent-config-reference`
+
+Basic Configuration
+===================
+
+The behavior of the master and agent can be controlled by setting configuration variables; this can
+be done using a configuration file, environment variables, or command-line options. Although values
+from different sources will be merged, we generally recommend sticking to a single source for each
+service to keep things simple.
```

Comment:
> Although values from different sources will be merged, we generally recommend sticking to a single source for each service to keep things simple.

My guess is that sysadmins don't need to be told this.

Instead, I would expect to see a statement about the relative priority of each one.  That's usually the statement I see in a manual that comes after "you can specify this way or that way".

Reply:
what is the priority of each one?

Reply:
resolved